README.md

```diff
@@ -346,6 +346,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [aasm.cr](https://github.com/veelenga/aasm.cr) - Easy to use finite state machine for Crystal classes
  * [accord](https://github.com/neovintage/accord) - Sharable validations for Crystal objects
  * [any_hash.cr](https://github.com/Sija/any_hash.cr) - Recursive Hash with better JSON::Any included
+ * [atomic_write.cr](https://github.com/chris-huxtable/atomic_write.cr)  -  Reduces the opportunity for file corruption during write or append
```

Comment:
There are two spaces before and after the dash

Reply:
Fixed